examples/gateway/proxy-car/README.md

```diff
@@ -0,0 +1,47 @@
+# Gateway as Proxy for Trustless CAR Remote Backend
+
+This is an example of building a Gateway that uses `application/vnd.ipld.car`
+responses from another gateway acting as a remote Trustless Gateway and IPNS resolver.
+
+**NOTE:** the CAR backend MUST implement [IPIP-0402: Partial CAR Support on Trustless Gateways](https://specs.ipfs.tech/ipips/ipip-0402/)
+
+## Build
+
+```bash
+> go build -o gateway
+```
+
+## Usage
+
+First, you need a compliant gateway that supports both [CAR requests](https://www.iana.org/assignments/media-types/application/vnd.ipld.car) and IPNS Record response
+types. Once you have it, run the proxy gateway with its address as the host parameter:
+
+```
+./gateway -g https://trustless-gateway.link -p 8040
+```
+
+### Subdomain gateway
+
+Now you can access the gateway in [localhost:8040](http://localhost:8040). It will
```

Comment:
```suggestion
Now you can access the gateway in [`localhost:8040`](http://localhost:8040/ipfs/bafybeiaysi4s6lnjev27ln5icwm6tueaw2vdykrtjkwiphwekaywqhcjze). It will
```